### ISSUE TITLE
ModuleNotFoundError: No module named 'pip.req'

### ISSUE THREAD

**Issue by vduseev** [onfido/s3-uploader#1]
## Issue
While installing using Pipenv following error appears:
```
$ pipenv install s3-uploader 
Creating a virtualenv for this project…
Using /Users/user/.pyenv/versions/3.6.5/bin/python3.6 (3.6.5) to create virtualenv…
⠋Already using interpreter /Users/user/.pyenv/versions/3.6.5/bin/python3.6
Using base prefix '/Users/vduseev/.pyenv/versions/3.6.5'
New python executable in /Users/user/.local/share/virtualenvs/test-s3-tools-hkPt1nKT/bin/python3.6
Also creating executable in /Users/user/.local/share/virtualenvs/test-s3-tools-hkPt1nKT/bin/python
Installing setuptools, pip, wheel...done.

Virtualenv location: /Users/user/.local/share/virtualenvs/test-s3-tools-hkPt1nKT
Creating a Pipfile for this project…
Installing s3-uploader…
Collecting s3-uploader
  Downloading https://files.pythonhosted.org/packages/3f/e2/5f11573baf1e0efca74d04c1e9f1c71f3a5e53a45741f9071ce32776b202/s3-uploader-0.0.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/lb/87rs8s1s5hv8764v3c71g1zc0000gn/T/pip-install-y02w3oub/s3-uploader/setup.py", line 5, in <module>
        from pip.req import parse_requirements
    ModuleNotFoundError: No module named 'pip.req'

    ----------------------------------------

Error:  An error occurred while installing s3-uploader!
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/lb/87rs8s1s5hv8764v3c71g1zc0000gn/T/pip-install-y02w3oub/s3-uploader/
```

## Thoughts
Here is a [nice description of the same error](https://stackoverflow.com/questions/25192794/no-module-named-pip-req) on Stack Overflow.

**Comment by avihoo** [comment]
Hi @vduseev,

Thanks for posting this issue.
I can confirm this is an issue as I was able to reproduce this locally.

Will update you as soon as this it gets fixed.

Thanks,
Avihoo

**Comment by avihoo** [comment]
I've raised a PR with a fix for the issue.

https://github.com/onfido/s3-uploader/pull/2

Will release a new version to PyPi soon.

**Comment by avihoo** [comment]
I've just uploaded a new version (0.0.6) to PyPi with the fix.

Let me know if this works and the issue can be resolved.

**Comment by vduseev** [comment]
Thanks for the incredibly fast fix @avihoo, I can confirm that the issue is fixed in the new 0.0.6 version.
